collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -314,6 +316,14 @@ public void onClick(View v) {
         }
     }
 
+    @Override
+    protected void onStart() {
+        super.onStart();
+        if (mapView != null) {
+            mapView.onStart();
```

Comment:
I'm just wondering if it's possible to put the `MapView` in a `Fragment` so we can have all the lifecycle "forwarding" in there. Thinking we have like a `MapBoxInitializationFragment` that the `MainMenuActivity` can host. Then we can keep all this noise out of `MainMenuActivity`.

Reply:
I think it's a good idea and I did it please take a look.